ginipaybank/src/doc/source/capture-features.rst

```diff
@@ -0,0 +1,280 @@
+            message = when (validationError) {
+                FileImportValidator.Error.TYPE_NOT_SUPPORTED -> ...
+                FileImportValidator.Error.SIZE_TOO_LARGE -> ...
+                FileImportValidator.Error.TOO_MANY_PDF_PAGES -> ...
+                FileImportValidator.Error.PASSWORD_PROTECTED_PDF -> ...
+                FileImportValidator.Error.TOO_MANY_DOCUMENT_PAGES -> ...
+            }
+        }
+        AlertDialog.Builder(this)
+            .setMessage(message)
+            .setPositiveButton("OK") { _, _ -> finish() }
+            .show()
+    }
+
+    fun startGiniPayBankSDKForImportedFile(importedFileIntent: Intent) {
+        // Configure capture first
+        configureCapture();
+
+        fileImportCancellationToken = 
+            GiniPayBank.startCaptureFlowForIntent(captureImportLauncher, this, importedFileIntent)
+    }
+
+Component API
+^^^^^^^^^^^^^
+
+When using the Component API you need to create a ``Document`` from the intent using
+``GiniPayBank.createDocumentForImportedFiles()`` and then continue either to the ``ReviewFragmentCompat``,
+``MultiPageReviewFragment`` or ``AnalysisFragmentCompat``:
+
+.. code-block:: java
+
+    fun startGiniPayBankSDKForImportedFile(importedFileIntent: Intent) {
+        GiniPayBank.createDocumentForImportedFiles(importedFileIntent, this, object : AsyncCallback<Document, ImportedFileValidationException> {
+            override fun onSuccess(result: Document) {
+                if (result.isReviewable) {
+                    // If you have enabled capturing documents with multiple pages:
+                    launchMultiPageReviewScreen()
+                    // OR if you didn't:
+                    launchSinglePageReviewScreen()
+                } else {
+                    launchAnalysisScreen(result)
+                }
+                finish()
+            }
+
+            override fun onError(exception: ImportedFileValidationException) {
+                handleFileImportError(exception)
+            }
+
+            override fun onCancelled() {
+                // Handle cancellation.
+            }
+        })
+    }
+
+Return Assistant
+----------------
+
+The return assistant features allows your users to view and edit the payable items in an invoice. The total amount is
+updated to be the sum of only those items which the user opts to pay.
+
+To enable this feature simply set ``returnAssistantEnabled`` to ``true`` in the ``CaptureConfiguration``: 
+
+.. code-block:: java
+
+    CaptureConfiguration(returnAssistantEnabled = true)
+
+Screen API
+~~~~~~~~~~~
+
+When integrating using the Screen API it is enough to enable the return assistant feature. The Gini Pay Bank SDK will
+show the return assistant automatically if the invoice contained payable items and will update the extractions returned
+to your app according to the user's changes.
+
+The ``amountToPay`` extraction is updated to be the sum of items the user decided to pay and also includes discounts or
```

Comment:
I would stop after `decided to pay`. `It includes discounts and additional charges that might be present on the document`.